Casks/s/skychart.rb

```diff
@@ -10,9 +10,8 @@
   homepage "https://www.ap-i.net/skychart/"
 
   livecheck do
-    url "https://sourceforge.net/projects/skychart/rss?path=/1-software/"
-    regex(/skychart[._-]v?(\d+(?:.\d+)+)[._-]x86[._-]64[._-]macosx\.dmg/i)
-    strategy :page_match
+    url "https://sourceforge.net/projects/skychart/rss?path=/1-software"
+    regex(%r{url=.*?/skychart[._-]v?(\d+(?:.\d+)+)[._-]x86[._-]64[._-]macosx\.dmg}i)
```

Comment:
```suggestion
    regex(%r{url=.*?/skychart[._-]v?(\d+(?:[.-]\d+)+[a-z]?)[^"' >]*?\.dmg}i)
```

I'm not a fan of wildcard delimiters when we know what we're looking to match. In this case, `[.-]` will handle something like `4.2.1-4073` (or `1.2.3`).

As above, the most recent release is from 2019 and there's only an Intel dmg, so I would omit the architecture suffix here as well. We can also gloss over the `-macosx` suffix, as dmg files are intended for macOS anyway. The same caveat applies here but we can always tighten this up if/when it surfaces an unstable version.